js/web/script/pull-prebuilt-wasm-artifacts.ts

```diff
@@ -145,12 +145,14 @@ downloadJson(
                 extractFile(zip, WASM_FOLDER, 'ort-wasm-simd-threaded.wasm', folderName);
                 extractFile(zip, WASM_FOLDER, 'ort-wasm-simd.jsep.wasm', folderName);
                 extractFile(zip, WASM_FOLDER, 'ort-wasm-simd-threaded.jsep.wasm', folderName);
+                extractFile(zip, WASM_FOLDER, 'ort-training-wasm-simd.wasm', folderName);
 
                 extractFile(zip, JS_FOLDER, 'ort-wasm.js', folderName);
                 extractFile(zip, JS_FOLDER, 'ort-wasm-threaded.js', folderName);
                 extractFile(zip, JS_FOLDER, 'ort-wasm-threaded.worker.js', folderName);
                 extractFile(zip, JS_FOLDER, 'ort-wasm-simd.jsep.js', folderName);
                 extractFile(zip, JS_FOLDER, 'ort-wasm-simd-threaded.jsep.js', folderName);
+                extractFile(zip, JS_FOLDER, 'ort-training-wasm-simd.js', folderName);
```

Comment:
Training should have their own artifact

Reply:
Ah, what do you mean? Training does have its own artifacts: ort-training-wasm-simd.js and ort-training-wasm-simd.wasm

Reply:
@guschmue do you mean that we should have different artifacts in the CI rather than the existing "Debug_wasm" and "Release_wasm" ( for example, "Debug_training_wasm" and "Release_training_wasm" ) ?

Reply:
our normal build doesn't produce ort-training-wasm-simd.js and ort-training-wasm-simd.wasm I assume how would they end up in the artifact?
Or do we always produce those ?

Reply:
Sorry, should've added this in the description for context -- a couple days ago, [this PR](https://github.com/microsoft/onnxruntime/pull/16521) was merged in, which extends the WASM API wrapper to include training functions + modifies the CMake to generate ort-training-wasm-simd artifacts. This PR modifies the pipeline to produce the ort-training artifacts which will end up in the zip.

Reply:
so we are now always building 8 wasm files ?

Reply:
7, since there's no need for a threaded version of training yet. The only additional artifact is ort-training-wasm-simd.wasm, but in the future, there will most likely be an ort-training-wasm-simd-threaded.wasm.

Reply:
guess its ok for now